tools/check-style.sh

```diff
@@ -11,72 +11,60 @@
 # 5. Missing space between right parenthesis and brace, e.g. 'for (...){'
 # 6. opening brace on its own line. It should always be on the same line as the
 #    if/while/for/do statment.
-# 
+#
 # Invoke as: tools/check-style.sh
 #
 
-errors=0
+check_style_errors=0
 IFS=$'\n'
-found=
-# The mt=41 sets a red background for matched tabs:
-GREP_COLORS='mt=41' GREP_COLOR='41' grep $'\t' include tests/*.{cpp,py,h} docs/*.rst -rn --color=always |
-        while read f; do
-    if [ -z "$found" ]; then
-        echo -e '\033[31m\033[01mError: found tabs instead of spaces in the following files:\033[0m'
-        found=1
-        errors=1
-    fi
 
-    echo "    $f"
-done
-
-found=
-grep -IUlr $'\r' include tests/*.{cpp,py,h} docs/*.rst --color=always |
-        while read f; do
-    if [ -z "$found" ]; then
-        echo -e '\033[31m\033[01mError: found CRLF characters in the following files:\033[0m'
-        found=1
-        errors=1
-    fi
-
-    echo "    $f"
-done
+found="$( GREP_COLORS='mt=41' GREP_COLOR='41' grep $'\t' include tests/*.{cpp,py,h} docs/*.rst -rn --color=always )"
+if [ ! -z "$found" ]; then
+    # The mt=41 sets a red background for matched tabs:
+    echo -e '\033[31m\033[01mError: found CRLF characters in the following files:\033[0m'
+    check_style_errors=1
+    echo "$found" | while read line; do
+        echo -e '\033[31m\033[01mError: found tabs instead of spaces in the following files:\033[0m'
```

Comment:
This looks like a copy-and-paste error: the echo inside the read loop should just be `echo "    $line"`.  However I think it would be easier to just replace the entire
```bash
    echo "$found" | while read line; do
        echo "    $line"
    done
```
with just:
```bash
    echo "$found" | sed -e 's/^/    '
```

The following blocks print the right thing, but would still benefit from the same one-line `sed` simplification.

Reply:
Added, though needed: 
 ```bash
echo "$found" | sed -e 's/^/    /'
```

Reply:
Whoops, yeah, accidentally left out the last `/`.